.maintain/bench-check/hydradx-bench-data.json

```diff
@@ -1,23 +1,23 @@
 {
-    "amm": {
-        "create_pool": 189.0,
-        "add_liquidity": 171.0,
-        "remove_liquidity": 170.0,
-        "sell": 122.0,
-        "buy": 121.0
+    "xyk": {
+        "create_pool": 174.0,
+        "add_liquidity": 143.0,
+        "remove_liquidity": 139.0,
+        "sell": 102.0,
+        "buy": 102.0
     },
     "exchange": {
-        "sell_intention": 109.0,
-        "buy_intention": 112.0,
-        "sell_extrinsic": 121.0,
-        "on_finalize_for_one_sell_extrinsic": 163.0,
-        "buy_extrinsic": 120.0,
-        "on_finalize_for_one_buy_extrinsic": 179.0
+        "sell_intention": 102.0,
+        "buy_intention": 142.0,
```

Comment:
Is really buy that much more expensive?

Reply:
nope. mistake.  it is fixed. it is the same as sell.